File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
	    public int solution(int n, String control) {
	        int answer = 0;
	        
	        char[] c = control.toCharArray();
	        
	        for(char cc : c){
                if(cc == 'w'){
                    n += 1;
                }else if(cc == 's'){
                    n -= 1;
                }else if(cc == 'd'){
                    n += 10;
                }else if(cc == 'a'){
                    n -= 10;
                }             
	        }
	        answer = n;
            
	        return answer;
	    }
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.py
def solution(n, control):
    answer = n
    
    for char in control:
        answer += updateValue(char)
    
    return answer

def updateValue(key):
    if key == 'w':
        return 1
    elif key == 's':
        return -1
    elif key == 'd':
        return 10
    elif key == 'a':
        return -10
    else:
        return 0
